pkg/exporters/stackdriver/stackdriver_exporter.go

```diff
@@ -30,7 +30,7 @@ import (
 	"google.golang.org/api/option"
 	"k8s.io/klog/v2"
 
-	"github.com/avast/retry-go"
+	"github.com/avast/retry-go/v4"
 	"k8s.io/node-problem-detector/pkg/exporters"
```

Comment:
I'm wondering if import statements can be sorted using goimports

Reply:
Probably not so hard. There's also a need to verify during automated testing.